Adapter/PhpcrOdmAdapter.php

```diff
@@ -154,7 +169,21 @@ public function createAutoRoute(UriContext $uriContext, $contentDocument, $autoR
      */
     public function createRedirectRoute(AutoRouteInterface $referringAutoRoute, AutoRouteInterface $newRoute)
     {
-        $referringAutoRoute->setRedirectTarget($newRoute);
+        switch ($this->redirectTarget) {
+            case self::REDIRECT_CONTENT:
+                $target = $newRoute->getContent();
```

Comment:
imho, we should make redirect to content the default, but fall back to the route if there is no content (which is a valid situation). i don't think we need the configuration options with constants, if there is a content of a route and it implements RouteReferrersReadInterface, we should take that over the route. but only in that case. the content could be not route aware, or there can be routes with no content object.